presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -639,6 +659,19 @@ existing data in S3::
       external_location = 's3://my-bucket/data/logs/'
     )
 
+Performance
```

Comment:
Similarly, let's remove these two section headers, as I don't think it makes sense to break up the examples this way. The examples section is a list of various things you can do with the Hive connector, that may or may not be (well) documented elsewhere. It's also the quick reference section used to remember how to do something.